tests/make_summary.py

```diff
@@ -292,11 +300,15 @@ def FormatPlots(base_dir, test_instances):
 
 def main(argv):
   base_dir = argv[1]
+  output_file = open(argv[2], 'w')
```

Comment:
The convention I've been following is to have HTML output on stdout.  And then if you need logging, there is a function log() that does print >>sys.stderr, so that logging doesn't get mixed with HTML.  But I guess this is OK for now.